components/skin/ui/src/main/resources/PhenoTips/Skin.xml

```diff
@@ -429,6 +429,129 @@ document.observe('xwiki:suggest:containerCreated', function(event) {
   </object>
   <object>
     <name>PhenoTips.Skin</name>
+    <number>3</number>
```

Comment:
Since this is attached to the skin, it affects not just the language switch, but also all other inline forms (families, studies, workgroups, users...). As such, I'd commit this with its own issue number.